plugin/trino-hive/src/main/java/io/trino/plugin/hive/orc/OrcFileWriterFactory.java

```diff
@@ -134,7 +135,8 @@ public Optional<FileWriter> createFileWriter(
             OptionalInt bucketNumber,
             AcidTransaction transaction,
             boolean useAcidSchema,
-            WriterKind writerKind)
+            WriterKind writerKind,
+            DateTimeZone parquetTimeZone)
```

Comment:
I'd like to know why is it that we need `parquetTimeZone` in the context of ORC related classes?


Same question for `RcFileFileWriterFactory`.

Please consider changing `ParquetFileWriterFactory` in such a fashion that the `HiveFileWriterFactory` interface doesn't need to be changed to contain `parquetTimeZone` parameter.

Reply:
Removed it from here now